src/requires.jl

```diff
@@ -96,4 +96,25 @@ function __init__()
                                   histogram.weights)
         end
     end
+
+    @require Measurements="eff96d63-e80a-5855-80a2-b1b0885c5ab7" begin
+        function PGFPlotsX.Coordinates(x::AbstractVector{T}, y::AbstractVector;
+                                            kwargs...) where T <: Measurements.Measurement
```

Comment:
would it make sense to restrict to `Real` measurements here, and in the other 2 methods?

Reply:
Yes, that makes sense. Should be fixed now.